src/mqt/bench/benchmarks/grover.py

```
@@ -11,18 +11,19 @@
 
 import numpy as np
-from qiskit.circuit import AncillaRegister, QuantumCircuit, QuantumRegister
+from qiskit.circuit import AncillaRegister, ForLoopOp, QuantumCircuit, QuantumRegister
 from qiskit.circuit.library import grover_operator
 
 from ._registry import register_benchmark
 
 
 @register_benchmark("grover", description="Grover's Algorithm")
-def create_circuit(num_qubits: int) -> QuantumCircuit:
+def create_circuit(num_qubits: int, for_loop: bool = False) -> QuantumCircuit:
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)